README.md

```diff
@@ -12,11 +12,11 @@ Join us in creating a browser, messenger, and gateway to a decentralized world.
 
 ## Why?
 
-We believe in a medium of pure free trade, economies with fair, permission-less access and a world without intermediaries. We want to create policies that can exist between friends or scale globally, we want to communicate securely and be uninhibited by legacy systems.
+We believe in a medium of pure free trade, economies with fair, permission-less access, and a world without intermediaries. We want to create policies that can exist between friends or scale globally, we want to communicate securely and be uninhibited by legacy systems.
```

Comment:
Hey @ayushrakesh 
Thanks for your PR!

This usage, with the comma before the "and," is often called the "Oxford comma"
It's typically used in lists of three or more items. 

However, when used in a list of only two items, it can add emphasis, or sometimes clarity, to the distinction between the two items.

It is worth noting that there aren't any grammatical errors that you've corrected by proposing this change.
ccing docs team on this : @jorge-campo

Reply:
@ayushrakesh, these are only grammatical mistakes if you adhere to the American English spelling rules, which we don't follow.

We do not use Oxford commas in our documents or copy, any appearance of them is likely a typo.

Thanks.